core/src/main/scripts/importer/validateData.py

```diff
@@ -981,8 +981,10 @@ class MutationsExtendedValidator(Validator):
 
     NULL_AA_CHANGE_VALUES = ('', 'NULL', 'NA')
 
+    # extra unofficial Variant classification values from https://github.com/mskcc/vcf2maf/issues/88:
+    EXTRA_VARIANT_CLASSIFICATION_VALUES = ['Splice_Region', 'Splice_Site']
```

Comment:
@pieterlukasse Sorry I was not clear; `Splice_Site` is an official classification, but `Splice_Region` is not. Can you remove `Splice_Site` from `EXTRA_VARIANT_CLASSIFICATION_VALUES`? It is already is still (correctly) in `VARIANT_CLASSIFICATION_VALUES`.